modules/protect.php

```diff
@@ -666,11 +666,24 @@ function protect_call( $action = 'check_ip', $request = array () ) {
 			$request['multisite'] = get_blog_count();
 		}
 
+
+		/**
+		 * Filter controls maximum timeout in waiting for reponse from protect servers
```

Comment:
In the past, we've tried to add periods at the end of each sentence in filter descriptions and param descriptions.